src/components/modals/TransactionAdvanceDetails.js

```diff
@@ -26,6 +26,7 @@ const feeString = {
 
 type OwnProps = {
   bridge: AirshipBridge<null>,
+  deviceName: string,
```

Comment:
Why do you need to pass the deviceName as a prop and not just call "getDeviceName" in the modal?